communities/src/lib.rs

```diff
@@ -106,9 +105,6 @@ pub mod pallet {
 			community_metadata
 				.validate()
 				.map_err(|_| <Error<T>>::InvalidCommunityMetadata)?;
-			if let Some(i) = nominal_income {
-				validate_nominal_income(&i).map_err(|_| <Error<T>>::InvalidNominalIncome)?;
-			}
```

Comment:
Validate only validated if the value is negative; type safety gives us this guarantee now already.